dlp/snippets/inspect/inspect_bigquery.go

```diff
@@ -85,28 +85,28 @@ func inspectBigquery(w io.Writer, projectID string, infoTypeNames []string, cust
 	// Create a PubSub Client used to listen for when the inspect job finishes.
 	pubsubClient, err := pubsub.NewClient(ctx, projectID)
 	if err != nil {
-		return fmt.Errorf("pubsub.NewClient: %w", err)
+		return fmt.Errorf("pubsub.NewClient: %v", err)
 	}
 	defer pubsubClient.Close()
 
 	// Create a PubSub subscription we can use to listen for messages.
 	// Create the Topic if it doesn't exist.
 	t := pubsubClient.Topic(pubSubTopic)
 	if exists, err := t.Exists(ctx); err != nil {
-		return fmt.Errorf("t.Exists: %w", err)
+		return fmt.Errorf("t.Exists: %v", err)
 	} else if !exists {
 		if t, err = pubsubClient.CreateTopic(ctx, pubSubTopic); err != nil {
-			return fmt.Errorf("CreateTopic: %w", err)
+			return fmt.Errorf("CreateTopic: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.